acceptance-tests/tests/workflows/cmsTemplateFlow.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { describe, beforeAll, it, expect, afterAll } from 'vitest';
+import { TestState } from '../../lib/TestState';
+import { ENTER } from '../../lib/prompt';
+
+const TEMPLATE = {
+  name: 'test-template',
+  folder: 'test-template.html',
+};
+
+describe('CMS Template Flow', () => {
```

Comment:
This flow tests:
1. creates a template locally
2. Uploads that template to the Design Manager
3. Lists the contents of the the Design Manager and makes sure that the uploaded file is in there
4. removes the file from the design manager
5. Lists the contents again to make sure the file is gone from Design Manager

so this tests `hs create template`, `hs upload`, `hs list`, and `hs remove`